### DISCUSSION TITLE
Direct/retrograde observations of planetary motions in new framework

### DISCUSSION THREAD

**Discussion by Carrickview** [Stellarium/stellarium#3109] [Ideas]
https://sol24.net/data/html/SOHO/C3/96H/VIDEO/

The stars change position parallel to the orbital plane as a function of the Earth's orbital motion and minus any influence of daily rotation. This is due to observations seen from a satellite with its focus on the central Sun where the satellite tracks with the Earth around our central star.

https://www.theplanetstoday.com/

The relative position of the Earth and Mercury on the opposite side of the Sun, while scrolling the dates forward, gives observers a clear perspective of Mercury currently moving behind the Sun in direct motion, whereas in a number of weeks, it will be seen travelling in the opposite direction between the Earth and the Sun.

A new framework is necessary insofar as when viewing the SOHO timelapse, everything to the right of the Sun is registered on Earth as a morning appearance and to the left of the Sun as a twilight or evening appearance. Mercury is therefore transitioning from morning to twilight appearance.

The new framework is not unknown as it is a more accurate description of the oldest framework for timekeeping based on the Egyptian Decans and the heliacal risings or first seasonal appearance of the stars responsible for the calendar system and ultimately RA/Dec software-

".. on account of the procession of the rising of Sirius by one day in the course of 4 years,.. therefore it shall be, that the year of 360 days and the 5 days added to their end, so one day shall be from this day after every 4 years added to the 5 epagomenae before the new year" Canopus Decree 238 BC

It is a rather large project and any queries or contentions can be addressed in a civil manner.

**Comment by gzotti** [comment]
I don't understand what kind of new framework this would need. You can just 
- switch off topocentric corrections, 
- center view on the sun, 
- switch off atmosphere, 
- go into equatorial mode, 

and see Mercury in orbit around the sun.

**Comment by Carrickview** [comment]
The first Sun-centred astronomers did not distinguish between the direct/retrograde motions of all the planets depending on whether they move faster or slower than the Earth, and they still do not make that distinction-

" Now what is said here of Jupiter is to be understood of Saturn and Mars also. In Saturn, these retrogressions are somewhat more frequent than in Jupiter because its motion is slower than Jupiter's, so the Earth overtakes it in a shorter time. In Mars they are rarer, its motion being faster than that of Jupiter so that the Earth spends more time in catching up with it. Next, as to Venus and Mercury, whose circles are included within that of the Earth, stoppings and retrograde motions appear in them also, due not to any motion that really exists in them but to the annual motion of the Earth. This is acutely demonstrated by Copernicus . . .

You see, gentlemen, with what ease and simplicity the annual motion -- if made by the Earth -- lends itself to supplying reasons for the apparent anomalies which are observed in the movements of the five planets. . . . It removes them all and reduces these movements to equable and regular motions, and it was Nicholas Copernicus who first clarified for us the reasons for this marvellous effect."
  Galileo 1632, Dialogue Concerning the Two Chief World Systems

The Ptolemaic framework which these researchers used was based on ecliptic longitude, where the planets 'wandered' against a line drawn by the direct motion of the Sun through the constellations-

http://astro.dur.ac.uk/~ams/users/solar_year.gif

https://apod.nasa.gov/apod/ap011220.html

"Moreover, we see the other five planets also retrograde at times, and
stationary at either end [of the regression]. And whereas the sun
always advances along its own direct path, they wander in various
ways, sometimes straying to the south and sometimes to the north; that
is why they are called "planets" [wanderers]. Copernicus


A new framework is necessary to account for the back-and-forth or direct/retrograde motions of Venus and Mercury-

https://www.youtube.com/watch?v=w2uCtot1aDg

**Reply by gzotti** [reply]
You can simulate all that and illustrate your extensive collection of citations with the existing program as outlined above, which closely replays the good old SOHO videos on YouTube. (Just seen from the geocenter, not SOHO in L1.)

If you want to show a Ptolemaic world view or illustrate epicyclic movement, that would require another program, to be developed by other people in their free time. Of course it would be "nice to have" to show the planets computed according to the Toledan, Alfonsine, Prutenian and Rudolfine tables and a few other medieval models. This could be done in a dedicated plugin that then shows markers more or less close to the planets according to modern computation.

**Reply by Carrickview** [reply]
The change in the position of the stars relative to the central Sun is a new demonstration of the Earth's orbital motion so that we see the motions of the faster-moving Venus and Mercury almost in their raw state with the central Sun at the centre of the motion of those two planets and ours.

https://www.youtube.com/watch?v=w2uCtot1aDg

The slower-moving Mars, Jupiter and Saturn require a different framework because the faster-moving Earth overtakes those slower-moving planets, causing them to fall behind in view from the perspective of a stationary field of stars-

https://apod.nasa.gov/apod/ap011220.html

It is a highly technical matter, and not everyone can see the distinctions, as I have come to understand over 30 years with this topic.

It doesn't help that contemporaries still imagine that Kepler's description of the direct/retrograde motion of Mars is geocentric despite his method stating it is a combination of the Earth's orbital motion and Mars-

https://en.wikipedia.org/wiki/Johannes_Kepler#/media/File:Kepler_Mars_retrograde.jpg

"Copernicus, by attributing a single annual motion to the earth, entirely rids the planets of these extremely intricate coils, leading the individual planets into their respective orbits, quite bare and very nearly circular. In the period of time shown in the diagram, Mars traverses one and the same orbit as many times as the 'garlands' you see looped towards the centre, with one extra, making nine times, while at the same time, the Earth repeats its circle sixteen times " Kepler Astronomia Nova 1609

It is a tangled topic presently for those who are unfamiliar with the conversion of the Ptolemaic system into a heliostatic system using a moving Earth and whether the planets move faster or slower than we do. The Ptolemaic system falls short as no hypothesis is needed with the SOHO imaging. The C3 camera fixes its gaze on the stationary Sun, obviating the need to infer a central Sun at the centre of all planetary motions, however a new framework is necessary to contain that perspective.

**Comment by Carrickview** [comment]
It is one of those dismaying features of this era that those engaged in solar system observations of the planets and moons in a Sun-centred system are unable to adapt to the framework where no hypothesis is needed apart from the observation that the stars change from an evening to morning appearance relative to the foreground and central Sun as a function of the Earth's orbital motion.

In the matter of direct/retrograde motions, it is a scaled-up version of Galileo's observations of Jupiter's satellites to a planetary scale where Venus and Mercury are seen to orbit our parent star as seen from the slower-moving Earth-

https://www.youtube.com/watch?v=FcrBAuLBXag

https://www.youtube.com/watch?v=w2uCtot1aDg

To facilitate the emergence of a new framework that is a variation on the first system of observations that generated the 365/366-day calendar system, a number of major modifications are required to the perspectives of researchers like Galileo, Kepler and Copernicus.

Although technical at this level of consideration, there is a warmth to the endeavour in revisiting observations that haven't been touched since the 16th and early 17th century. It is made possible by an orbiting satellite, timelapse footage from the surface and many of contemporary tools, along with knowledge of the original Egyptian framework that was not available to the great astronomers.

With over three decades of experience, it can be a lonely affair; however, it is curiosity and the possibilities of this new approach which should motivate observers to join the discussion without fear and that unfamiliarity is perfectly acceptable.

**Comment by Carrickview** [comment]
The central theme of this new framework is that the stars are seen to change position parallel to the orbital plane as a function of the Earth's orbital motion. With Jupiter presently coming into range of the C3 camera as it transitions from a recent twilight appearance to an upcoming dawn appearance, it helps observers define the orbital plane and something more productive, although that can wait.

https://sol24.net/data/html/SOHO/C3/96H/VIDEO/


Jupiter's actual motion is from right to left gauged against the background stars but unlike Mercury which recently transitioned to an evening appearance in its motion from right to left, Jupiter moves slower than the Earth it will be seen changing position with the stars as a reflection of the Earth's own motion.

The framework which permits today's observers to put the motions of the planets in the context of a central Sun is the same one by which the 365/366-day calendar system is constructed, which in turn provides the basis for the Ptolemaic system on which Copernicus based his hypothesis for a moving Earth in a Sun-centred system. The framework was unknown to Copernicus and Galileo and resolves an issue that nobody could at the time of the Galileo affair.

It would be exceptional if an individual other than myself had a firm grasp on what is being presented so that while mathematical or experimental theorists are determined to avoid the new framework through which timekeeping and planetary dynamics are combined by means of a tracking satellite, that avoidance cannot last.

It appears that my visit here to this discussion forum has reached a dead-end on that account.

**Reply by Carrickview** [reply]
It is so strange to hear my presentation for splitting the direct/retrograde resolutions for planetary motions in two and depending on whether the planets are moving faster or slower than the Earth.  My involvement in this astronomical endeavour is entirely voluntary so, even though it is over a decade since I explained a distinction that neither Copernicus, Kepler nor Galileo made due to a lack of observational data now possible with a tracking satellite, I have not put a hindrance before the present community in accepting the resolution few know is even there.

https://www.youtube.com/watch?v=VXQh1xFce7s&t=568s

About 8;30 minutes in, the celebrity divides the perspectives in two without the slightest indication that the original astronomers did not make the distinction nor how the conclusion was drawn recently.

" What appears in the planets as [the alternation of] retrograde and direct motion is due, not to their motion, but to the Earth's. The motion of the Earth alone, therefore, suffices [to explain] so many apparent irregularities in the heaven." Copernicus, Commentariolus

https://copernicus.torun.pl/en/ressources/archives/copernicus-minor-astronomical-writings/commentariolus

It was the hypotheses based on the Ptolemaic framework that led to the introduction of the mutation of Brahe as they could not resolve the direct/retrograde of Venus and Mercury sufficiently using that framework hence Venus and Mercury orbit the Sun while the Sun and the rest of the planets orbit the Earth -

https://assets.sutori.com/user-uploads/image/3c3fd02d-8e73-418b-a2f3-ed31b47dcd22/c635f778dfc37a942103af0f1b202acf.gif

The introduction of the new framework where the Earth's orbital motion causes the annual change in the position of the stars as opposed to a stationary field of background stars used by Ptolemy resolves the issue by splitting the perspectives in two.

One perspective is accounting for the direct/retrogrades of the slower-moving planets seen from a faster-moving Earth using a fixed stellar background as a gauge where the Earth overtakes Jupiter and Saturn in the following time-lapse causing them to fall behind in view-

https://apod.nasa.gov/apod/ap011220.html

The other perspective is that the majority of the Earth's orbital motion sets the Sun up as a central reference so the direct/retrograde motions of Venus and Mercury represent their back-and-forth motion around our parent star as seen from our slower-moving Earth-

https://www.youtube.com/watch?v=w2uCtot1aDg

I do not mind that someone else explains the distinction, however, I would like students to understand the explanation properly and the journey through the historical and technical details which make the conclusion enjoyable as a visual narrative and an excursion through history. Dull people will not appreciate the importance of what this perspective represents although others do, even and without proper attribution.
